source/includes/harvest/_job_posts.md

```diff
@@ -7,20 +7,27 @@ Describes the online job posts for an organization's jobs (as seen on the Job Bo
 ```json
 {
   "id": 123,
-  "title": "Button Pusher",
-  "location": {
-    "name": "The Island"
-  },
+  "title": "Software Engineer",
+    "location": {
+      "id": 123456,
+      "name": "New York, NY",
+      "office_id": null,
+      "job_post_custom_location_id": 303145,
+      "job_post_location_type": {
+        "id": 1,
+        "name": "Free Text"
+      }
+    },
```

Comment:
There is an extra indentation with this field
```suggestion
  "location": {
    "id": 123456,
    "name": "New York, NY",
    "office_id": null,
    "job_post_custom_location_id": 303145,
    "job_post_location_type": {
      "id": 1,
      "name": "Free Text"
    }
  },
```

Reply:
This is now updated!